e2e/features/step_definitions/other-party-subscription.steps.ts

```diff
@@ -0,0 +1,98 @@
+
+    await anyCcdPage.clickElementById('subscriptions_appointeeSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.setValueByElementId('subscriptions_appointeeSubscription_tya', 'appointee123')
+    await anyCcdPage.setValueByElementId('subscriptions_appointeeSubscription_email', 'appointee-test@mailinator.com')
+    await anyCcdPage.setValueByElementId('subscriptions_appointeeSubscription_mobile', '01234567890')
+    await anyCcdPage.clickElementById('subscriptions_jointPartySubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.setValueByElementId('subscriptions_jointPartySubscription_tya', 'jointParty123')
+    await anyCcdPage.setValueByElementId('subscriptions_jointPartySubscription_email', 'jointparty-test@mailinator.com')
+    await anyCcdPage.setValueByElementId('subscriptions_jointPartySubscription_mobile', '01234567890')
+
+    await anyCcdPage.clickElementById('subscriptions_supporterSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.setValueByElementId('subscriptions_supporterSubscription_tya', 'supportParty123')
+    await anyCcdPage.setValueByElementId('subscriptions_supporterSubscription_email', 'supportparty-test@mailinator.com')
+    await anyCcdPage.setValueByElementId('subscriptions_supporterSubscription_mobile', '01234567890')
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartySubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartySubscription_tya', 'otherParty123')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartySubscription_email', 'otherparty-test@mailinator.com')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartySubscription_mobile', '01234567890')
+     await anyCcdPage.clickElementById('otherParties_0_otherPartySubscription_subscribeEmail_' + action);
+     await anyCcdPage.clickElementById('otherParties_0_otherPartySubscription_subscribeSms_' + action);
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyAppointeeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyAppointeeSubscription_tya', 'otherPartyAppointee123')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyAppointeeSubscription_email', 'otherpartyAppontee-test@mailinator.com')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyAppointeeSubscription_mobile', '01234567890')
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyRepresentativeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyRepresentativeSubscription_tya', 'otherParty123')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyRepresentativeSubscription_email', 'otherparty-test@mailinator.com')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyRepresentativeSubscription_mobile', '01234567890')
+ } else {
```

Comment:
The No action is not being used. Is it nice to have this code?

Reply:
Added missing test case to set subscription to No